strapi/strapi
# enhancement: add preview page header

### What does it do?

Adds a header to the preview page. It displays the main field, the document's status, and a button to copy the preview page link (within Strapi Admin, not the direct preview URL).

I found that the CM's document status wasn't using translations, so I changed it. The modified key is new but for draft and publish it should be compatible with translations from v4

The draft/publish toggle will be in another ticket/PR

### How to test it?

Open preview for content types where it's enabled. Check the mainfield and document status, try changing them too. And copy the page URL
